src/bci_build/package/spack.py

```diff
@@ -19,26 +21,15 @@
         os_version=os_version,
         is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
         logo_url="https://spack.io/assets/images/spack-logo-white.svg",
-        version="%%spack_version%%",
+        version=(spack_pkg_version := get_pkg_version("spack", os_version)),
         version_in_uid=False,
         additional_versions=["%%spack_minor%%"],
         replacements_via_service=[
-            Replacement(
-                regex_in_build_description="%%spack_version%%",
```

Comment:
why are we removing that here? this is unrelated to the README stuff ?

Reply:
The replacement isn't used anywhere else, so I dropped it to not run the service needlessly

Reply:
but're still having https://github.com/SUSE/BCI-dockerfile-generator/commit/488575b4d70e43991df3f7e033aa4547ec1c157d#diff-e769e1c8493dbb658288f74a0b9aa816607c37dc429a6f7d5651ea02f56cbb6aR19 ? so %%spack_minor%% needs to be changed as well?

Reply:
ok I see this doesn't have to be replaced strictly speaking.